partiql-types/src/main/kotlin/org/partiql/value/datetime/impl/OffsetTimeHighPrecision.kt

```diff
@@ -86,4 +95,10 @@ internal class OffsetTimeHighPrecision private constructor(
                 }
             }
         }
+
+    private fun normalizeElapsedTime(timePassed: BigDecimal): BigDecimal {
+        val maxBD = BigDecimal.valueOf(MAX_ELAPSED_SECOND)
+        val mod = timePassed % maxBD
+        return if (mod < BigDecimal.ZERO) maxBD + mod else mod
+    }
```

Comment:
Also, with the example above where `timePassed` (negative or positive) essentially results in the next or previous day, can this be documented? AKA something like what LocalTime says: "This wraps around midnight.".

Reply:
`%` is actually not modulo, it is technically remainder in java...
So for `-3 % 100`, the result will be `-3`.
I will rename the variable to avoid confusion.